### PR TITLE
[LIB-352] quickfix - update for object with files

### PR DESCRIPTION
**Pull request by Zhebr**

### CODE REVIEW

File: syncano/connection.py [hunk 1]
```diff
@@ -234,7 +234,12 @@ def make_request(self, method_name, path, **kwargs):
         :raises SyncanoValueError: if invalid request method was chosen
         :raises SyncanoRequestError: if something went wrong during the request
         """
-        files = kwargs.get('data', {}).pop('files', None)
+        data = kwargs.get('data', {})
+
+        files = {k: v for k, v in data.iteritems()
+                 if hasattr(v, 'read')}
```
Comment (opalczynski) [inline syncano/connection.py]: Well. In such case we can have in files some object which is not a file at all.

It rather be rare situation - but can happen.

Comment (Zhebr) [inline syncano/connection.py]: Agree. However, I treat it as quickfix. Don't know faster way to determine if object is file-like.

Comment (opalczynski) [inline syncano/connection.py]: OK :) So lgtm, but create task to handle it properly :)

Comment (dancio) [inline syncano/connection.py]: lgtm

Comment (Zhebr) [inline syncano/connection.py]: @opalczynski task created for next sprint

**Pull request merged**